packages/office-addin-lint/package.json

```diff
@@ -33,7 +33,6 @@
   },
   "devDependencies": {
     "@types/assert": "^1.4.2",
-    "@types/commander": "^2.12.2",
```

Comment:
same question here about whether this is intentionally removed

Reply:
same answer.  :-)